pages/common/perl.md

```diff
@@ -0,0 +1,23 @@
+# perl
+
+> The Perl 5 language interpreter 'CLI'.
```

Comment:
The 'CLI' is confusing me. Is that just a reference to the fact that this is a command line tool?